src/compiler/moduleNameResolver.ts

```diff
@@ -901,11 +900,19 @@ export interface NonRelativeModuleNameResolutionCache extends NonRelativeNameRes
     getOrCreateCacheForModuleName(nonRelativeModuleName: string, mode: ResolutionMode, redirectedReference?: ResolvedProjectReference): PerModuleNameCache;
 }
 
+/** @internal */
+export interface MissingPackageJsonInfo {
+    packageDirectory: string;
+    contents: boolean;

```

Comment:
```suggestion
    directoryExists: boolean;
```

?

Reply:
I kept the name because it becomes easier to check for things that come out of cache with same property names but I guess I can add helper for isPackageJsonInfo ?